tests/components/notify/test_html5.py

```diff
@@ -97,18 +91,17 @@ def test_registering_new_device_view(self):
         """Test that the HTML view works."""
         hass = MagicMock()
 
-        with tempfile.NamedTemporaryFile() as fp:
-            hass.config.path.return_value = fp.name
-            fp.close()
+        m = mock_open()
+        with patch('homeassistant.components.notify.html5.open', m):
             service = html5.get_service(hass, {})
 
             assert service is not None
 
             # assert hass.called
-            assert len(hass.mock_calls) == 3
+            assert len(hass.mock_calls) == 4
```

Comment:
Actually, I figured it out and updated the commit, but it doesn't seem to have updated on this PR yet.  Maybe because of the GitHub outage that just happened?